test/options.js

```diff
@@ -0,0 +1,86 @@
+var test = require('tape');
+var Graylog = require('..');
+
+test('options', function (t) {
+	t.throws(function () {
```

Comment:
Tape doesn't have describe() like mocha does. But you can make sub-tests. In any case, more important I think that I add messages to all assertions to make the test suite output more readable.